micro_sam/sample_data.py

```diff
@@ -21,7 +23,8 @@ def fetch_image_series_example_data(save_directory: Union[str, os.PathLike]) ->
     os.makedirs(save_directory, exist_ok=True)
     print("Example data directory is:", save_directory.resolve())
     fname = "image-series.zip"
-    unpack_filenames = [os.path.join("series", f"im{i}.tif") for i in range(3)]
+    # use first two files for image series (thrid file is not the same shape)
+    unpack_filenames = [os.path.join("series", f"im{i}.tif") for i in range(2)]
```

Comment:
I included the image of a different shape on purpose, because the image series annotator supports that.
Is there a particular reason you removed it here?

Reply:
Oh, I see. Thank you for explaining!
How do you want these to be opened in napari, as three separate layers?
Or I could put them all in the same layer if I pad with zeros to make them the same shape.

napari does not support different shaped image frames in the same layer well. Each napari layer contains a `.data` numpy array, and numpy does not support joining different array shapes like that (the [awkward array](https://awkward-array.org/doc/main/) project does support this, but I don't think anyone has ever tried to combine it with napari - I suspect it would be difficult)

Reply:
The series annotation tool loads these images one at a time. The idea is to enable users to process multiple images from a folder more conveniently.
Check out https://github.com/computational-cell-analytics/micro-sam/blob/master/examples/image_series_annotator.py to run the demo for it.

Reply:
And if it's a problem for napari that the image shapes are different than let's skip the plugin for this example data. I am not really sure if it makes sense at all to provide the image series annotator as a plugin, so we can skip this data for now.

Reply:
Ok, I'll remove the image series from the sample datasets provided to napari.